generated/nidaqmx/system/device.py

```diff
@@ -1071,16 +1117,15 @@ def reset_device(self):
         self._interpreter.reset_device(
             self._name)
 
-    def restore_last_ext_cal_const(self, device_name):
+    def restore_last_ext_cal_const(self):
         """
-        
-
-        Args:
-            device_name (str): 
+        This function nullifies any self-calibration you perform on the
+        device. If you have never performed a self-calibration on the
+        device, this function has no effect.
         """
 
         self._interpreter.restore_last_ext_cal_const(
-            self._name, device_name)
+            self._name)
```

Comment:
This looks formatted wrong (I don't think this should be multiple lines), but I think the PR will yell at you

Reply:
I ran `poetry run ni-python-styleguide lint`, no files changed after running it.
And then I ran `poetry run black generated\nidaqmx\system\device.py`, it fixed the multiple line, however it also formatted other lines as well, eg: capitalize hex codes. Should I push these changes to this PR?

Snippet of the changes:
![image](https://github.com/ni/nidaqmx-python/assets/27721199/0979c25b-068e-4608-bd40-b98e27b8de79)

Full changes:
[device.py.diff.txt](https://github.com/ni/nidaqmx-python/files/14018707/device.py.diff.txt)

Reply:
`poetry run ni-python-styleguide lint` runs `black --check`. It doesn't modify any files, and it tolerates some formatting variations.

To reformat files, run `poetry run ni-python-styleguide fix`. Note that this only reformats files that it needs to, so it also tolerates some formatting variations.

Either way, files in the `generated` dir are excluded in both the black config and ni-python-styleguide config, so `poetry run black generated\nidaqmx\system\device.py` should have no effect.

Reply:
> This looks formatted wrong (I don't think this should be multiple lines), but I think the PR will yell at you

@zhindes It's codegen, so the PR doesn't check formatting.

The reason it's wrapped is because https://github.com/ni/nidaqmx-python/blob/master/src/codegen/templates/function_template.py.mako#L80 doesn't have a special case to fit all of the parameters on one line.

It's fine.

Reply:
The build checks are failing because the codegen files in the trunk don't match the output of the codegen script.

```
0s
Run git diff --exit-code
diff --git a/generated/nidaqmx/system/device.py b/generated/nidaqmx/system/device.py
index 92bba3f..545dd32 [1](https://github.com/ni/nidaqmx-python/actions/runs/7622732014/job/20761347475?pr=473#step:9:1)00644
--- a/generated/nidaqmx/system/device.py
+++ b/generated/nidaqmx/system/device.py
@@ -1,6 +1,5 @@
 # Do not edit this file; it was automatically generated.
 
-
 import deprecation
 
 from datetime import datetime
Error: Process completed with exit code 1.
```

Please rerun the codegen script and **don't** run black on the output. The output of the codegen script is fine.

Reply:
Ah, it's because I forgot to rerun codegen after I removed extra newline in the device.py.mako.

Reply:
Not checking codegen or modifying it for this specific corner case totally makes sense; my bad!